### ISSUE TITLE
v2.2.0 doesn't recognize "skipWords" option

### ISSUE THREAD

**Issue by yaeda** [nodaguti/textlint-rule-spellchecker#54]
## Issue
textlint-rule-spellchecker@2.2.0 doesn't recognize "skipWords" option.

I run textlint with this .textlintrc, but got errors with "funciton" and "errror".
```
{
  "rules": {
    "spellchecker": {
        "skipWords": ["funciton", "errror"]
    }
  }
}
```

## My Rough Investigation
I think the cause of this issue is in the building or publishing process.

Please install with `yarn add textlint-rule-spellchecker` then see/compare these two files.

- `node_modules/textlint-rule-spellchecker/src/spellchecker.js`
- `node_modules/textlint-rule-spellchecker/lib/spellchecker.js`

`lib/spellchecker.js` seems to be old (no logic for skipWords option).

## Environment
- node : v9.2.0
- npm: 5.6.0
- yarn: 1.3.2

Thank you.

**Comment by nodaguti** [comment]
Thanks for the report!

Indeed the published version of `textlint-rule-spellchecker` doesn't seem to have been updated after changing the code in `/src`.

I'm in a little confusion because I surely set `prepublish` to transpiling the code before publishing  to make sure this kind of mistake could never happen...

Anyway I've released the recompiled version as v2.2.1, so now you should be able to get the latest code.

Sorry for inconvenience!

**Comment by yaeda** [comment]
Thank you for re-publishing.
I confirmed it works correctory 😄
